devtools/x-core/src/lib.rs

```diff
@@ -29,7 +29,7 @@ pub struct XCoreContext {
     config: XCoreConfig,
     current_dir: PathBuf,
     current_rel_dir: PathBuf,
-    git_cli: GitCli,
+    git_cli: DebugIgnore<OnceCell<GitCli>>,
```

Comment:
Hmm any particular reason to make this `DebugIgnore`? For `PackageGraphPlus` the issue is that the `Debug` impl prints out a ton of info.

Reply:
Hmm, my expectation that a debugger could trigger the instantiation of the once_cell, I guess that was incorrect.